client/src/components/SpotEntryForm/SpotEntryForm.js

```diff
@@ -0,0 +1,63 @@
+import styled from "@emotion/styled";
+import React from "react";
+import RedButton from "../RedButton/RedButton";
+import { useForm } from "react-hook-form";
+import { createSpot } from "../../api/spots";
+import PropTypes from "prop-types";
+
+const EntryForm = styled.form`
+  label,
+  input,
+  textarea {
+    font-family: var(--font-lato);
+    margin: 0.5rem 0;
+    display: block;
+    width: 100%;
+  }
+
+  button {
+    margin-top: 1.2rem;
+  }
+`;
+
+const SpotEntryForm = ({ location }) => {
+  const { register, handleSubmit } = useForm();
+
+  const onSubmit = (data) => {
+    console.log(data);
+    try {
+      data.latitude = location.latitude;
+      data.longitude = location.longitude;
+      const spotCreated = createSpot(data);
+      console.log(spotCreated);
```

Comment:
I guess these logs are temporary?
If not, better remove or replace them with visible feedback :).

Reply:
True!

Reply:
I have the second console log there because if it is not finished yet and it tells me that it works.